api/commentables.rb

```diff
@@ -42,11 +42,15 @@
   
   thread.author = user
   thread.save
+
+  # Mark thread as read for owner user on creation
+  user.mark_as_read(thread)
```

Comment:
Should this follow the same pattern of following the user.subscribe() call below, after having done the thread error check?